src/components/Restoration/CorrectNameOptions.vue

```diff
@@ -81,17 +98,22 @@ import CorrectNameRequest from '@/components/Restoration/CorrectNameRequest.vue'
   }
 })
 export default class CorrectNameOptions extends Vue {
-  /** The options to display */
-  @Prop() readonly correctionNameChoices!: Array<string>
+  @Prop({ default: null }) readonly actionTxt!: string
+  @Prop({ default: () => [] }) readonly correctionNameChoices!: Array<NameChangeOptions>
+  @Prop({ required: true }) readonly businessId!: string
+  @Prop({ required: true }) readonly entityType!: CorpTypeCd
+  @Prop({ required: true }) readonly nameRequest!: NameRequestIF
+  @Prop({ required: true }) readonly companyName!: string
+  @Prop({ required: true }) readonly fetchAndValidateNr!: () => Promise<NameRequestIF>
```

Comment:
I decided to make these required instead of having defaults. An error is thrown if this component is called and the required props are not provided.

Reply:
This component is now in BCRS shared repo.